src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx

```diff
@@ -68,21 +46,21 @@ export const ItemFlow = ({ name, index, onRemove }: ItemFlowProps) => {
   return (
     <ToggleItemContainer
       title={title}
-      Content={Content}
-      HeaderContent={Actions}
+      Content={ItemFlowContent}
+      HeaderContent={ItemFlowActions}
       contentProps={{
-        items: conditions,
         name: itemName,
+        conditions,
         onRemove: handleRemoveCondition,
-        conditionalError,
+        'data-testid': dataTestid,
       }}
       headerContentProps={{
         name: itemName,
         onAdd: handleAddCondition,
         onRemove,
-        'data-testid': `builder-activity-item-flow-${index}`,
+        'data-testid': dataTestid,
```

Comment:
does it make sense?
```suggestion
        'data-testid': `${dataTestid}-header`,
```

Reply:
it is used just for the actions like Add/Remove/Collapse so there is no sense to separate header and content